test/functional/apps/management/_kibana_settings.js

```diff
@@ -46,6 +46,18 @@ export default function({ getService, getPageObjects }) {
     });
 
     describe('state:storeInSessionStorage', () => {
+      async function getStateFromUrl() {
```

Comment:
Please see comment which explains the change: https://github.com/elastic/kibana/pull/60087/files#r397250237

Reply:
Extracted to separate pr in scope of: https://github.com/elastic/kibana/issues/61230